pkg/controller/machine_util_test.go

```diff
@@ -992,7 +993,7 @@ var _ = Describe("machine_util", func() {
 
 				waitForCacheSync(stop, c)
 
-				Expect(testNode.Spec.Taints).Should(Equal(expectedNode.Spec.Taints))
+				Expect(reflect.DeepEqual(testNode.Spec.Taints, expectedNode.Spec.Taints)).To(Equal(true))
```

Comment:
Made this change to count for scenarios where slice ordering can differ. This was failing test cases occasionally.